yogstation/code/modules/guardian/standarrow.dm

```diff
@@ -153,7 +153,7 @@
 	INVOKE_ASYNC(src, .proc/get_stand, H, stats)
 
 /obj/item/stand_arrow/proc/get_stand(mob/living/carbon/H, datum/guardian_stats/stats)
-	var/list/mob/dead/observer/candidates = pollGhostCandidates("Do you want to play as the Guardian Spirit of [H.real_name]?", ROLE_HOLOPARASITE, null, FALSE, 100, POLL_IGNORE_HOLOPARASITE)
+	var/list/mob/dead/observer/candidates = pollGhostCandidates("Do you want to play as the Guardian Spirit of [H.real_name]?", ROLE_HOLOPARASITE, null, FALSE, 100, POLL_IGNORE_HOLOPARASITE, disallow_started_as_observer = TRUE)
```

Comment:
if you are really going with this holos should be allowed otherwise you won't be able to get ANY holos without killing someone first and hoping they agree to be yours, which is unikely